README.md

```diff
@@ -28,7 +28,7 @@ We require fully qualified schemas. You can find how the unqualified should be t
 Start the database, you can use docker for that in the following way:
 
 ```sh
-docker run --name mongo -p 4000:27017 -d mongo:6.0
+docker-compose up -d mongo mongo-replica-setup
```

Comment:
Use `docker compose` instead of `docker-compose`
https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose